styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -182,10 +183,11 @@
 
   private static final int REQUEST_CONCURRENCY = 32;
 
+  private final Closer closer = Closer.create();
```

Comment:
We can get `closer` from Apollo env, but passing it all the way down from `StyxScheduler` is awful. :(